lms/djangoapps/verify_student/models.py

```diff
@@ -569,15 +552,15 @@ def display_off(cls, user_id):
         property to false, so the notification banner can be switched off.
         """
         user = User.objects.get(id=user_id)
-        cls.objects.filter(user=user, status="denied").exclude(window=None).update(display=False)
+        cls.objects.filter(user=user, status="denied").update(display=False)
```

Comment:
I think we can get rid of `display_off()` entirely, since it was only ever used to toggle the reverification banner (which has now been removed)